lib/item_repository.rb

```diff
@@ -0,0 +1,56 @@
+require_relative './item'
+require_relative './order'
+
+class ItemRepository
+
+  def all
+    sql = 'SELECT id, name, price, quantity FROM items;'
+    result_set = DatabaseConnection.exec_params(sql, [])
+    items = []
+    result_set.each do |record|
+      items << unpack_record(record)
+    end
+    return items
+  end
+
+  def find(id)
+    sql = 'SELECT id, name, price, quantity FROM items WHERE id = $1;'
+    params = [id]
+    result_set = DatabaseConnection.exec_params(sql, params)
+    record = result_set[0]
+    unpack_record(record)
+  end
+
+  def create(item)
+    sql = 'INSERT INTO items (name, price, quantity) VALUES ($1, $2, $3);'
+    params = [item.name, item.price, item.quantity]
+    DatabaseConnection.exec_params(sql, params)
+    return nil
+  end
+
+  def delete(id)
```

Comment:
good display of delete method but i dont think we required it from the user story, same with find the method